README.md

```diff
@@ -110,6 +111,32 @@ module "service-scaling" {
 }
 ```
 
+## user-data
+Setup a ECS cluster and service user-data.
+
+### Available variables:
+ * [`project`]: String(optional): The project to tag EFS (default: "")
+ * [`environment`]: String(required):  For what environment is this cluster needed (eg staging, production, ...)
+ * [`cluster_name`]: String(required): The name of the ECS cluster.
+ * [`teleport_version`]: String(required): The version of Teleport to be used (default: 2.5.8).
+ * [`teleport_server`]: String(optional): the name of the teleport server to connect to(default: "")
+ * [`teleport_auth_token`]: String(optional): Teleport server node token (default: "")
```

Comment:
shouldn't `teleport_auth_token` and `teleport_server` be required and `teleport_version` optional? Only teleport version is set to a default